installer/x86_64/install-arch

```diff
@@ -77,6 +77,17 @@ init_onie_install()
     }
     . ./onie-blkdev-common
 
+    # wipe out current onie_dev if ONIE will embed to another
+    # one, or just keep updating to current one
+    curr_onie_dev=$(onie_get_boot_disk)
+    if [ -b "$curr_onie_dev" ] && [ "$curr_onie_dev" != "$onie_dev" ] ; then
+        if [ "$onie_boot_reason" = "embed" ] ; then
+            wipe_curr_onie_dev="yes"
+        else
+            onie_dev="$curr_onie_dev"
+        fi
+    fi
+
```

Comment:
Can you provide a bit more background?  It is not a lot of code, but it is a big change to way things work.

I think I understand, but can you please describe the intended use for this on the mailing list.  Not asking for a lot, but a short, high level description of the "requirements" and "expected use cases".